### PR TITLE
Add Revolver Ability & Polish a few things

### PR DESCRIPTION
**Pull request by armandorusso**
- Added Ganiel's Revolver Ability! Drag in bullets and shoot down Pimpkin heads!
See the Gif Below!

revolver_gif

Polish Adjustments I did while adding the ability:
- Fixed URP settings to improve anti-alliasing and shadows. Game looks a lot smoother now!
- Adjusted the damage of Confection. It was too powerful for the difficulty and cost
- Fixed the Confection Animation. It now plays at the end of the ability (just like how Revolver is)

Note: There is a known bug right now and we have no idea how to fix it. It doesn't detect the final pimpkin you hit. After hours of debugging, we aren't sure how to fix it (delaying the canvas still doesn't make it work for some reason). If you guys can figure it out, that would be awesome!

### CODE REVIEW

**Review by George540 — APPROVED** [review]
Looks good! Good teamwork. Animation and some delays may be changed later in the debug phase. I made an issue about it for later.

File: Assets/Scripts/Combat/Abilities/PimpkinHead.cs [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+using UnityEngine;
+
+public class PimpkinHead : MonoBehaviour
+{
+
+    private bool IsHit;
+    private RectTransform PimpkinBody;
+    private Vector3 StartingPosition;
+    private int Switch = 0;
+    private float randomIntX;
```
Comment (eli-lam) [inline Assets/Scripts/Combat/Abilities/PimpkinHead.cs:12]: Pascal case!

**Review by eli-lam — CHANGES_REQUESTED** [review]
Good job on the ability! Just a small change needed

File: Assets/Scripts/Combat/Abilities/PimpkinHead.cs (hunk 1 above)
Comment (armandorusso) [reply Assets/Scripts/Combat/Abilities/PimpkinHead.cs]: Fixed!

**Review by eli-lam — APPROVED** [review]
LGTM!

**Pull request merged**